### PR TITLE
Enterprise 4.1 monitoring and Telemetry final changes

### PR DESCRIPTION
**Pull request by rh-max**
Final changes for 4.1 for monitoring and Telemetry docs.

### CODE REVIEW

**Comment by rh-max** [comment]
@adellape Hey Alex, can you please do a peer review of this? It's a short one, and contains a change relevant for you.

File: monitoring/exposing-custom-application-metrics-for-autoscaling.adoc [hunk 1]
```diff
@@ -9,3 +9,7 @@ You can export custom application metrics for the horizontal pod autoscaler.
 
 include::modules/monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling.adoc[leveloffset=+1]
 
+.Additional resources
+
+See the xref:../nodes/pods/nodes-pods-autoscaling.adoc#nodes-pods-autoscaling[horizontal pod autoscaling documentation].
```
Comment (adellape) [inline monitoring/exposing-custom-application-metrics-for-autoscaling.adoc]: Could add a bullet to the front of this, but also I see in the built version of the guide, there's an `Additional resources` already at the end of the `monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling` module, which appears right before this:

```
For information about Horizontal Pod Autoscaler, see link:https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale[the Kubernetes documentation].
```

So the `Additional resources` are stacking. Maybe move the kube doc link to this assembly instead and have 2 bullets?

File: telemetry/opting-out-of-telemetry.adoc [hunk 2]
```diff
@@ -5,8 +5,15 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+ifdef::openshift-enterprise,openshift-origin[]
 You might need to opt out of Telemetry for your cluster. For example, you might need to comply with privacy laws.
+endif::[]
 
+ifdef::openshift-dedicated[]
+On OpenShift Dedicated, Telemetry is always enabled. You cannot opt out.
```
Comment (adellape) [inline telemetry/opting-out-of-telemetry.adoc]: s/OpenShift Dedicated/{product-title}/

**Review by adellape — CHANGES_REQUESTED** [review]
Couple nits, but otherwise looks good!

File: telemetry/opting-out-of-telemetry.adoc (hunk 2 above)
Comment (rh-max) [reply telemetry/opting-out-of-telemetry.adoc]: Isn't it "OpenShift Container Platform" but "OpenShift Dedicated"?

Comment (adellape) [reply telemetry/opting-out-of-telemetry.adoc]: Within an `openshift-dedicated`, `{product-title}` will render as "OpenShift Dedicated".

File: monitoring/exposing-custom-application-metrics-for-autoscaling.adoc (hunk 1 above)
Comment (rh-max) [reply monitoring/exposing-custom-application-metrics-for-autoscaling.adoc]: Good catch, done.

File: telemetry/opting-out-of-telemetry.adoc (hunk 2 above)
Comment (rh-max) [reply telemetry/opting-out-of-telemetry.adoc]: :man_facepalming: Done. If you the result looks good, can you please merge?

**Review by adellape — APPROVED** [review]
Changes LGTM!

Meant to say also, can you squash to 1 commit? Then good to merge.

**Comment by rh-max** [comment]
@adellape Great, thanks! Sure, just squashed.

**Pull request merged**